examples/RNN_toy_example.py

```diff
@@ -0,0 +1,143 @@
+"""
```

Comment:
You'll see that Scrutinizer already complained about this, the formatting for docstrings (including module-level ones) is:
```
"""A summary line.

A length explanation here.

"""
```
So you'll need a summary line, and a blank line before closing. Also make sure that the docstring never exceeds 75 characters.